aten/src/ATen/templates/CodegenUnboxingWrappers.cpp

```diff
@@ -0,0 +1,42 @@
+#include <torch/csrc/jit/runtime/operator.h>
+#include <torch/csrc/jit/runtime/custom_operator.h>
+#include <torch/csrc/jit/runtime/register_ops_utils.h>
+
+#include <ATen/CodegenFunctions.h>
+
+// ${generated_comment}
+
+// NOTE [Sharded File]: This file is generated in a sharded fashion to speed up
```

Comment:
You can probably kill this comment and replace it with `See Note [Sharded File]`

Can you also add a comment about this file? We probably want to explicitly mention that we're code-generating registrations to the JIT registry here, but they're currently only used by mobile builds.